GoogleForCreators/web-stories-wp
# Added video frame strip to the trim rail background

## Context

This implements #9104 as per the ticket description.

## Relevant Technical Choices

* I use the actual size, scale, and focal point of the video so the trim rail will only show the same frame snippets that you see in the actual video (e.g. if you zoom in on a small part, only that part of the video will be snapshotted for every frame). I'm not sure if this is standard though.

## Todo

* [ ] Should we display a loader while the strip generates?

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2021-10-29 at 00 29 44](https://user-images.githubusercontent.com/637548/139375882-9388856a-4909-4876-bd06-f9c762ae039a.png) | ![Screen Shot 2021-10-29 at 00 29 33](https://user-images.githubusercontent.com/637548/139375886-f97c30c9-1eb2-415e-9fe2-edf027ac6c1a.png) |

## Testing Instructions

This PR can be tested by following these steps:

1. Add a local video to a page
2. Enter trim mode
3. Observe that the trim rail background is a film strip from the video being trimmed

## Checklist




---

Fixes #9104
